### DISCUSSION TITLE
OneDrive网盘读取readme.md时报错错误500

### DISCUSSION THREAD

**Discussion by abbaccadd0** [AlistGo/alist#839] [Q&A]
RT，网盘是OneDrive，其他功能均正常，但Readme无法正常显示，报错信息为{"code":500,"message":"Get "...": dial tcp 13.107.136.9:443: i/o timeout","data":null}
请问这是如何产生的，要如何解决呢？
~~总不会是因为服务器连不上微软的服务器了吧~~

**Comment by xhofe** [comment]
根据报错信息，显然是这样的。

**Reply by abbaccadd0** [reply]
> 根据报错信息，显然是这样的。

您好，我对此仍然有疑问。我确定我的服务器可以正常连接到微软的服务器，它可以在服务器的浏览器中正常打开并下载，但是Alist中仍然提示错误500，问题出在哪里了呢？🤔

**Reply by xhofe** [reply]
日志就是连不上OneDrive文件服务器 超时了。其他不知道
